docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,175 @@
+# Autofill Manager
+
+The Tizen autofill framework allows more than one or more Autofill services to be installed on a Tizen device. Autofill manager is a module for managing the installed autofill services. Application developers can use autofill manager to get the current autofill service or change the autofill service.
+
+The main features of the Autofill Manager API include:
+
+- Installed autofill service list
+
+  You can [get the installed autofill service list](#installed-autofill-service-list) in your device.
+
+- Current autofill service
+
+  You can [get the current autofill service](#current-autofill-service).
+
+- Change of autofill service
```

Comment:
@mijin85cho and @imfine98 
Please check for the suggested change is correct:
Changing autofill service